### ISSUE TITLE
nixpkgs-update log analysis

### ISSUE THREAD

**Issue by henrirosten** [ryantm/nixpkgs-update#347]
While writing https://github.com/ryantm/nixpkgs-update/issues/346, I realized the existing logs at https://r.ryantm.com/log/ could be used to build interesting statistics for the nixpkgs-update failure reasons.

The idea would be to crawl the (latest) logs at https://r.ryantm.com/log/ and collect data for the nixpkgs-update failure reasons. This data could then be used to categorize the errors and attempt to answer questions such as:
- What are the most common nixpkgs-update failure reasons? Would it be possible to fix some of these in nixpkgs-update or in the packages themselves that fail to auto-update, to increase the number of packages that nixpkgs-update can update automatically?
- Are there packages where nixpkgs-update has not even tried to run the update in the past N weeks? If there are, what's the reason for that?

I'm writing this suggestion here, in case someone has already done something like this earlier.

**Comment by ryantm** [comment]
Nope, haven't done any work like that but would be interested in your findings.
